internal/utils/utils.go

```diff
@@ -126,6 +126,14 @@ func ConvertStringToTime(data string) (time.Time, error) {
 
 }
 
+func ConvertISO8601StringToTime(data string) (time.Time, error) {
+	dateTime, err := time.Parse("2006-01-02T15:04:05.000Z", data)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("unable to convert string to time: %s", err)
+	}
+	return dateTime, nil
```

Comment:
Here, this method returns two values i.e dateTime and `nil`.  Why Can't we return just dataTime value as second value will be `nil` always?

Reply:
It is not returning nil always, If you check line 132 we are returning error on it.

Reply:
Yes , it is returning error on that line. But at the end, you used a blank identifier (i.e `_` ) to capture second value at method calls. That means, it is not required, correct?

Will it be possible to return just value and see if we can log here directly an error? OR may be handle scenario what if there is an error?

Reply:
@patilsuraj767, captured and added log outside the method.